CWE/localisation/countries.csv

```diff
@@ -280,4 +280,6 @@ RBU_absolute_monarchy;The Kingdom of Russia and Belarus;;;;;;;;;x
 RBU_hms_government;The Kingdom of Russia and Belarus;;;;;;;;;x
 RBU_hms_government1;The Kingdom of Russia and Belarus;;;;;;;;;x
 RDA;North Afghanistan;;;;;;;;;x
-RDA_ADJ;Afghan;;;;;;;;;x
\ No newline at end of file
+RDA_ADJ;Afghan;;;;;;;;;x
+TRB;Triest - Zone B;;;;;;;;;x
```

Comment:
I think South Triest is better

Reply:
I've decided to edit this to South Trieste once this PR is merged. I'm not comfortable with giving tags names that are specific to a particular event chain when the tag can be released outside of the chain through normal indy rebels, dismantlements, liberate country CB etc. The name must be contextual to all possible scenarios.

Reply:
Ph yes sorry, the last chain was quite a brain drain and I totally forgot about it, I'll do it in the next push